plugins/obs-filters/noise-suppress-filter.c

```diff
@@ -331,8 +331,8 @@ static void *nvafx_initialize(void *data)
 }
 
 static inline void alloc_channel(struct noise_suppress_data *ng,
-				 uint32_t sample_rate, size_t channel,
-				 size_t frames)
+				 uint32_t sample_rate OBS_UNUSED,
```

Comment:
`sample_rate` is used on line 339 right below this.

Reply:
Only if SpeexDSP is enabled, if it's not it's unused and the compiler will complain - usually you'd use `maybe_unused` to satisfy the compiler, but that's not supported by all platforms, so using the default "UNUSED" attribute was cleaner.

Reply:
Sounds good.